symplyphysics/laws/electricity/transmission_lines/transmission_matrix_of_π_type_circuit.py

```diff
@@ -0,0 +1,100 @@
+## A - ratio of input voltage to output voltage at idle at the output,
+## B - ratio of input voltage to output current in case of a short circuit at the output,
+## C - ratio of input current to output voltage at idle at the output,
+## D - ratio of input current to output current in case of a short circuit at the output,
+## Z1 - first impedance,
+## Z2 - second impedance,
+## Z3 - third impedance.
+
+parameter_voltage_to_voltage = Symbol("parameter_voltage_to_voltage", dimensionless)
+parameter_impedance = Symbol("parameter_impedance", units.impedance)
+parameter_conductance = Symbol("parameter_conductance", units.conductance)
+parameter_current_to_current = Symbol("parameter_current_to_current", dimensionless)
+
+first_impedance = Symbol("first_impedance", units.impedance)
+second_impedance = Symbol("second_impedance", units.impedance)
+third_impedance = Symbol("third_impedance", units.impedance)
+
+
+law = Eq(Matrix([[parameter_voltage_to_voltage, parameter_impedance], [parameter_conductance, parameter_current_to_current]]),
+         Matrix([[1 + third_impedance / second_impedance, third_impedance], [(1 / first_impedance) + (1 / second_impedance) + third_impedance / (first_impedance * second_impedance), 1 + third_impedance / first_impedance]]))
+
+# This law might be derived via "transmission_matrix_for_a_series_load_in_line" law and
+# "transmission_matrix_for_a_parallel_load_in_line" law.
+
+parallel_law_applied_1 = parallel_law.law.subs({
+    parallel_law.load_impedance: first_impedance,
+})
+matrix_derived_1 = solve(parallel_law_applied_1, [parallel_law.parameter_voltage_to_voltage, parallel_law.parameter_impedance, parallel_law.parameter_conductance, parallel_law.parameter_current_to_current], dict=True)[0]
+matrix_derived_1[parallel_law.parameter_impedance] = matrix_derived_1[parallel_law.parameter_impedance].scale_factor
+matrix_derived_1 = Matrix([[matrix_derived_1[parallel_law.parameter_voltage_to_voltage], matrix_derived_1[parallel_law.parameter_impedance]], [matrix_derived_1[parallel_law.parameter_conductance], matrix_derived_1[parallel_law.parameter_current_to_current]]])
+
+series_law_applied = series_law.law.subs({
+    series_law.load_impedance: third_impedance,
+})
+matrix_derived_3 = solve(series_law_applied, [series_law.parameter_voltage_to_voltage, series_law.parameter_impedance, series_law.parameter_conductance, series_law.parameter_current_to_current], dict=True)[0]
+matrix_derived_3[series_law.parameter_conductance] = matrix_derived_3[series_law.parameter_conductance].scale_factor
+matrix_derived_3 = Matrix([[matrix_derived_3[series_law.parameter_voltage_to_voltage], matrix_derived_3[series_law.parameter_impedance]], [matrix_derived_3[series_law.parameter_conductance], matrix_derived_3[series_law.parameter_current_to_current]]])
+
+parallel_law_applied_2 = parallel_law.law.subs({
+    parallel_law.load_impedance: second_impedance,
```

Comment:
But in the description first and third resistors are parallel. Why does second resistor come into the parallel law?